src/button/button-group.scss

```diff
@@ -4,6 +4,7 @@
 
 @mixin iui-button-group {
   display: inline-flex;
+  align-items: center;
```

Comment:
`align-items: center;` was added to support the ellipsis & spinner when used within the paginator component.
![Screen Shot 2021-05-18 at 3 46 49 PM](https://user-images.githubusercontent.com/849817/118713721-1f8b3300-b7d7-11eb-918f-1ad6b3f144a8.png)

Reply:
Wait, the paginator uses button group?

Reply:
It wasn't before, but now it is. Honestly it's better this way.

Reply:
I did switch the paginator to use a button group for larger touch target areas.